src/charts/utilization-bar/utilization-bar-chart.component.js

```diff
@@ -40,6 +40,7 @@
  * has been reached, the used donut arc will be red.
  * @param {string=} threshold-warning The percentage usage, when reached, denotes a warning.  Valid values are 1-100. When the warning threshold
  * has been reached, the used donut arc will be orange.
+ * @param {function(items)} tooltipFunction A passed in tooltip function which can be used to overwrite the default tooltip behavior
```

Comment:
I can foresee applications desiring different tooltips for used vs available. Thinking we should allow for availableTooltipFunction and usedTooltipFunction